packages/client/hmi-client/src/api/api.ts

```diff
@@ -3,12 +3,11 @@ import axios, { AxiosError, AxiosHeaders } from 'axios';
 import { EventSource } from 'extended-eventsource';
 import { ServerError } from '@/types/ServerError';
 import { Ref, ref } from 'vue';
-import { activeProjectId } from '@/composables/activeProject';
 import useAuthStore from '../stores/auth';
 
 export class FatalError extends Error {}
 
-function getProjectIdFromUrl(): string | null {
+export function getProjectIdFromUrl(): string | null {
```

Comment:
Should this be used by the composable `activeProject` instead of being imported to all services calls?